llvm/lib/Target/NVPTX/NVPTXTargetMachine.cpp

```diff
@@ -64,6 +64,11 @@ static cl::opt<bool> UseShortPointersOpt(
         "Use 32-bit pointers for accessing const/local/shared address spaces."),
     cl::init(false), cl::Hidden);
 
+static cl::opt<bool> EarlyByValArgsCopy(
```

Comment:
Perhaps copy/paste the PR description on top of this flag?  Seems like very useful context.

Reply:
Done.